cdap-app-fabric/src/main/java/io/cdap/cdap/app/runtime/ProgramStateWriter.java

```diff
@@ -47,6 +47,16 @@ void start(ProgramRunId programRunId, ProgramOptions programOptions, @Nullable S
    */
   void running(ProgramRunId programRunId, @Nullable String twillRunId);
 
+  /**
+   * Updates the program run's status to be stopping
```

Comment:
shall we add description similar to `start` method pointing to the ProgramRunStatus#STOPPING?